scripts/build-mp.sh

```diff
@@ -40,6 +40,4 @@ WD=$(pwd)
  cp build-WROVER_16M-OTA_USER/partition_table/partition-table.bin "${WD}/images" &&
  cp build-WROVER_16M-OTA_USER/bootloader/bootloader.bin "${WD}/images" &&
  cp build-WROVER_16M-OTA_USER/ota_data_initial.bin "${WD}/images" && cd "${WD}" &&
```

Comment:
I don't think the ending && cd "${WD}" && should be needed. The "(" ")" mean do all this in a sub-command and we should be back in the WD when this is all done.

Did you see some other behaviour?

Reply:
sorry  should have tested everything before I opened the pull request ... but  made changes and went threw it ... reflected n the changes in the pull